CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [Next release]:
```

Comment:
Did we change this text from TBD?

Reply:
Ah right, we use TBD on iOS. Let me change it